lizmap/modules/admin/templates/maps.tpl

```diff
@@ -21,7 +21,7 @@
                 <dd>
                     <table class="table">
                         {assign $section = 'repository:'.$repo->getKey()}
-                        {assign $properties = lizmapRepository::getProperties()}
+                        {assign $properties = $repo->getRepoProperties()}
```

Comment:
Why not using `$repo->getProperties()` and create `$repo->getRepoProperties()`

Reply:
Because getProperties is a static method that can be used out of lizmapRepository so we don't have to create a lizmapRepository instance just to access its list of properties which is always the same.

Reply:
@laurentj is it normal static methods can't be called in templates ?

Reply:
Yes we can call static methods in theory. But here, since we've got the repository object, we can call directly the method on the object.